### PR TITLE
Add protected variables

### LINKED ISSUE #473: Docker Secrets partially broken
According to the documentation of the docker image, the following environment variables can be passed as a docker secret by appending `_FILE` at the end of the environment variable:
- `IOTA_AUTH_USER`
- `IOTA_AUTH_PASSWORD`
- `IOTA_AUTH_CLIENT_ID`
- `IOTA_AUTH_CLIENT_SECRET`
- `IOTA_MONGO_USER`
- `IOTA_MONGO_PASSWORD`
- `IOTA_MQTT_KEY`
- `IOTA_MQTT_USERNAME`
- `IOTA_MQTT_PASSWORD`
- `IOTA_AMQP_USERNAME`
- `IOTA_AMQP_PASSWORD`

The docker secrets functionality for the first six variables (AUTH and MONGO) is implemented within the underlying iotagent-node-lib and works properly. However, the docker secrets functionality for the MQTT and AMQP variables is broken due to the following commit:

https://github.com/telefonicaid/iotagent-ul/commit/6938d883bbb539dd24e267504bf102f4d11b06ae

Handling the docker secrets for the MQTT and AMQP variables has been dropped without removing the variables from the related part of the documentation or implementing an approach similar to the one present within iotagent-node-lib.

### PR DESCRIPTION
**Pull request by jason-fox**
Fixes #473

### CODE REVIEW

**Comment by fgalan** [comment]
Similar PR needed in other IOTAs repositories? Or the problem is only in IOTA-UL one?

**Comment by jason-fox** [comment]
> Similar PR needed in other IOTAs repositories? Or the problem is only in IOTA-UL one?

According to the `README.md` file associated to the various docker containers, only Ultralight and JSON extend the standard list of protected variables.

File: lib/configService.js [hunk 1]
```diff
@@ -112,15 +113,26 @@ function processEnvironmentVariables() {
     ];
     const httpVariables = ['IOTA_HTTP_HOST', 'IOTA_HTTP_PORT', 'IOTA_HTTP_TIMEOUT', 'IOTA_HTTP_KEY', 'IOTA_HTTP_CERT'];
 
-    for (let i = 0; i < environmentVariables.length; i++) {
-        let value = process.env[environmentVariables[i]];
+    const protectedVariables = [
```
Comment (fgalan) [inline lib/configService.js:116]: Should an entry about the changes to be added in CHANGES_NEXT_RELEASE file?

Comment (fgalan) [reply lib/configService.js:116]: (Same for the "twin" PR)

Comment (jason-fox) [reply lib/configService.js:116]: Fixed 68616bc

**Review by fgalan — APPROVED** [review]
LGTM

**Comment by fgalan** [comment]
Fail in v10 and v14 execution seems to be the usual glitch. Merging anywya.

**Pull request merged**